README.md

```diff
@@ -1,4 +1,4 @@
-Akka [![Latest version](https://index.scala-lang.org/akka/akka/akka-actor/latest.svg)](https://index.scala-lang.org/akka/akka/akka-actor)[![Build Status](https://travis-ci.org/akka/akka.svg?branch=master)](https://travis-ci.org/akka/akka) 
+Akka [![Latest version](https://index.scala-lang.org/akka/akka/akka-actor/latest.svg)](https://index.scala-lang.org/akka/akka/akka-actor)[![Build Status](https://travis-ci.com/akka/akka.svg?branch=master)](https://travis-ci.com/akka/akka)
```

Comment:
```suggestion
Akka [![Latest version](https://index.scala-lang.org/akka/akka/akka-actor/latest.svg)](https://index.scala-lang.org/akka/akka/akka-actor)[![Build Status](https://travis-ci.com/akka/akka.svg?branch=master)](https://travis-ci.com/github/akka/akka)
```

Reply:
:-o is that `github/` new?

Reply:
not very new, but the old url redirects (though breaking the back button IIRC)